packages/eth-deposit/scripts/exec-viaInbox.js

```diff
@@ -1,15 +1,16 @@
 const { utils, providers, Wallet } = require('ethers')
-const { Inbox__factory, Bridge, networks } = require('arb-ts')
+const { EthBridger, getL2Network } = require ('arb-ts')
 const { parseEther } = utils
 const { arbLog, requireEnvVariables } = require('arb-shared-dependencies')
+const { IInbox__factory } = require('arb-ts/dist/lib/abi')
```

Comment:
not sure whats going on here, but this import is probably `null` since there is [no more index file](https://unpkg.com/browse/arb-ts@2.0.0-beta.13/dist/lib/abi/) inside `lib/abi`

we can either remove it or use `require('arb-ts/dist/lib/abi/factories/IInbox__factory')`

Reply:
updated